demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -212,6 +218,95 @@ def is_image_path_valid(self) -> bool:
             return False
         return True
 
+    def verify_readme_image_paths(self) -> bool:
+        """ Validate readme (not pack readme) images relative and absolute paths.
+
+        Returns:
+            bool: True If all links both relative and absolute are valid else False.
+        """
+        # If there are errors in one of the following validations return False
+        if any([self.check_readme_relative_image_paths(),
+                self.check_readme_absolute_image_paths()]):
+            return False
+        return True
+
+    def check_readme_relative_image_paths(self, is_pack_readme: bool = False) -> list:
+        """ Validate readme images relative paths.
+            (1) prints an error if relative paths in the pack README are found since they are not supported.
+            (2) Checks if relative paths are valid (in other readme files).
+
+        Arguments:
+            is_pack_readme (bool) - True if the the README file is a pack README, default: False
+
+        Returns:
+            list: List of the errors found
+        """
+        error_list = []
+        should_print_error = not is_pack_readme  # print error if its not pack readme
+        relative_images = re.findall(r'(\!\[.*?\])(\((?!http).*?\))', self.readme_content, re.IGNORECASE)
+        if relative_images:
+            if is_pack_readme:  # Check whether the README type is pack README which do not support relative paths.
+                for img in relative_images:
+                    try:
+                        relative_path = img[1]
+                    except IndexError:
+                        continue
+                    if relative_path:
+                        error_message, error_code = Errors.pack_readme_image_relative_path_error(img[0] + relative_path)
+                        formatted_error = self.handle_error(error_message, error_code, file_path=self.file_path,
+                                                            should_print=should_print_error)
+                        error_list.append(formatted_error)
+
+            else:  # Not pack readme relative paths are allowed but have to be valid.
+                for img in relative_images:
+                    try:
+                        relative_path = img[1]
+                    except IndexError:
+                        continue
+                    # playbook doc generator automatically adds 'Insert the link to your image here' to the
+                    # readme image path, which we want to skip.
```

Comment:
why skip? it should also fail the validation

Reply:
Done